schema/contact-lists.yml

```diff
@@ -0,0 +1,51 @@
+contact_list:
+  list_name:
+    - string
+    - required
+    - name of the contact list
+    - example: 'my_list'
+  description:
+    - text
+    - optional
+    - description of the contact list
+    - example: 'my_list description'
+  public_outlets_count:
+    - integer
+    - optional
+    - number of public outlets in the contact list
+    - example: '1'
+  private_outlets_count:
+    - integer
+    - optional
+    - number of private outlets in the contact list
+    - example: '1'
+  public_contacts_count:
+    - integer
+    - optional
+    - number of public contacts in the contact list
+    - example: '1'
+  private_contacts_count:
+    - integer
+    - optional
+    - number of private contacts in the contact list
+    - example: '1'
+  contact_list_type:
+    - string
+    - required
+    - the source system of the contact list
+    - example: 'GORKANA | NGC3'
```

Comment:
Can we add  `C3` to the `contact_list_type` as we hope to begin testing and supporting C3 migrations by the end of Q1.